ChangeLog.md

```diff
@@ -18,6 +18,8 @@ Note: due to API changes, this release may not work correctly with node versions
 - From protocol version 8, raw command `GetPoolStatus` indicates if a validator is suspended and,
   if it is in the current committee, if it is primed for suspension and the current count of
   missed rounds.
+- From protocol version 8 `account show` also shows if a validator is suspended.
+- Add command `pool status` for getting information about a pool.
```

Comment:
Since we've already released 8.0.0, when you merge up, make sure this goes under `Unreleased` and not under `8.0.0`.